### PR TITLE
reviewing

### PR DESCRIPTION
**Pull request by gloomy-ghost**
push alternative wording suggestions to this branch, and discussion can be started by commenting in the diff.

### CODE REVIEW

File: ForEveryone.Net.ass [hunk 1]
```diff
@@ -998,7 +998,7 @@ Dialogue: 0,0:23:58.76,0:24:00.06,Chinese,,0,0,0,,-这不符合逻辑
 Dialogue: 0,0:24:00.10,0:24:01.73,English,,0,0,0,,because we have one road system.
 Dialogue: 0,0:24:00.10,0:24:01.73,Chinese,,0,0,0,,我们只有一个交通系统
 Dialogue: 0,0:24:01.76,0:24:02.80,English,,0,0,0,,It's really important that you can go
-Dialogue: 0,0:24:01.76,0:24:02.80,Chinese,,0,0,0,,能自由行动，从任何地方到任何地方
+Dialogue: 0,0:24:01.76,0:24:02.80,Chinese,,0,0,0,,能自由行动，从任何地方到任何地方（！）
```
Comment (gloomy-ghost) [inline ForEveryone.Net.ass]: 时间轴：CPS爆炸

File: ForEveryone.Net.ass [hunk 2]
```diff
@@ -452,7 +452,7 @@ Dialogue: 0,0:10:45.00,0:10:47.43,Chinese,,0,0,0,,我就能把这些插头全转
 Dialogue: 0,0:10:52.70,0:10:55.70,English,,0,0,0,,"Suppose all the information stored on computers everywhere
 Dialogue: 0,0:10:52.70,0:10:55.70,Chinese,,0,0,0,,“假设存储在计算机上的所有信息
 Dialogue: 0,0:10:55.73,0:10:57.43,English,,0,0,0,,were linked," I thought.
-Dialogue: 0,0:10:55.73,0:10:57.43,Chinese,,0,0,0,,都有关联，”我想
+Dialogue: 0,0:10:55.73,0:10:57.43,Chinese,,0,0,0,,都有关联”，我想（～）
```
Comment (gloomy-ghost) [inline ForEveryone.Net.ass]: 句子有点长，标点有点乱

File: ForEveryone.Net.ass [hunk 3]
```diff
@@ -134,41 +134,41 @@ Dialogue: 0,0:02:47.33,0:02:51.83,Chinese,,0,0,0,,我觉得那会是……会是
...
 Dialogue: 0,0:03:38.56,0:03:41.06,English,,0,0,0,,Once you've got a computer
-Dialogue: 0,0:03:38.56,0:03:41.06,Chinese,,0,0,0,,一旦你有一台电脑
+Dialogue: 0,0:03:38.56,0:03:41.06,Chinese,,0,0,0,,只要有一台电脑
 Dialogue: 0,0:03:41.10,0:03:43.06,English,,0,0,0,,what it does is up to your imagination.
 Dialogue: 0,0:03:41.10,0:03:43.06,Chinese,,0,0,0,,它做什么完全取决于你的想象
 Dialogue: 0,0:03:43.10,0:03:44.40,English,,0,0,0,,So that is the point.
 Dialogue: 0,0:03:43.10,0:03:44.40,Chinese,,0,0,0,,这是重点
 Dialogue: 0,0:03:44.43,0:03:47.80,English,,0,0,0,,But there was this magic threshold when you had a computer.
-Dialogue: 0,0:03:44.43,0:03:47.80,Chinese,,0,0,0,,但那时的电脑也有一个神奇的门槛
+Dialogue: 0,0:03:44.43,0:03:47.80,Chinese,,0,0,0,,但那时的电脑也有一个神奇的门槛（？）
```
Comment (gloomy-ghost) [inline ForEveryone.Net.ass:172]: 没太get到原意

File: ForEveryone.Net.ass [hunk 4]
```diff
@@ -344,69 +344,69 @@ Dialogue: 0,0:08:21.80,0:08:24.06,Chinese,,0,0,0,,我们提供所有的设施，
...
+Dialogue: 0,0:09:33.23,0:09:36.00,Chinese,,0,0,0,,他异常地擅长融会贯通
 Dialogue: 0,0:09:36.03,0:09:38.60,English,,0,0,0,,It's fabulously wired up, but it's not very common
-Dialogue: 0,0:09:36.03,0:09:38.60,Chinese,,0,0,0,,实在难以置信，要知道这是不太寻常的……
+Dialogue: 0,0:09:36.03,0:09:38.60,Chinese,,0,0,0,,不只是普通意义上的擅长，很难碰到
 Dialogue: 0,0:09:38.63,0:09:41.46,English,,0,0,0,,to find people who think kind of...
-Dialogue: 0,0:09:38.63,0:09:41.46,Chinese,,0,0,0,,找到能想到他所需的事物的人……
+Dialogue: 0,0:09:38.63,0:09:41.46,Chinese,,0,0,0,,有人的思考模式会如此得……
 Dialogue: 0,0:09:41.50,0:09:44.16,English,,0,0,0,,I don't know... all at once.
-Dialogue: 0,0:09:41.50,0:09:44.16,Chinese,,0,0,0,,我不知道……一次性地
+Dialogue: 0,0:09:41.50,0:09:44.16,Chinese,,0,0,0,,怎么说呢…一次性包含全部
```
Comment (gloomy-ghost) [inline ForEveryone.Net.ass]: 一步到位？

File: ForEveryone.Net.ass (hunk 1 above)
Comment (jyhi) [reply ForEveryone.Net.ass]: 在考虑拆分

File: ForEveryone.Net.ass (hunk 3 above)
Comment (jyhi) [reply ForEveryone.Net.ass:172]: 这句话本来就比较模糊，我估计 Tim 想说的是“虽然电脑能做任何事情，但因为那时的电脑没有连接起来，它们的作用很有限”；这之后的片段也在讲 Connection 相关的概念，所以我感觉像是在表达某种未能突破的极限。

**Comment by jyhi** [comment]
Since I accidentally ignored this PR, I will perform a manual merge :)

File: ForEveryone.Net.ass (hunk 1 above)
Comment (jyhi) [reply ForEveryone.Net.ass]: @gloomy-ghost  “能自由穿梭于各地”如何

**Comment by gloomy-ghost** [comment]
I already merged your commits into this branch. GitHub doesn't show it, but there should be no problem to use the normal merge

File: ForEveryone.Net.ass (hunk 1 above)
Comment (gloomy-ghost) [reply ForEveryone.Net.ass]: lgtm

File: ForEveryone.Net.ass (hunk 3 above)
Comment (gloomy-ghost) [reply ForEveryone.Net.ass:172]: 有没有可能是顺着前文的“只要有台电脑就无所不能”，“但首先要有台电脑才能打开新世界的大门”

**Comment by jyhi** [comment]
> but there should be no problem to use the normal merge

There are still some translations that I don't agree with; I am also fixing problems you noted right in the subtitles. We can have a further discussion after the manual merge (which appears to be clearer in diff).

File: ForEveryone.Net.ass [hunk 5]
```diff
@@ -363,7 +363,7 @@ Dialogue: 0,0:09:07.63,0:09:09.46,Chinese,,0,0,0,,在说一个句子的时候…
 Dialogue: 0,0:09:09.50,0:09:12.10,English,,0,0,0,,start a sentence, and then by the time he got...
 Dialogue: 0,0:09:09.50,0:09:12.10,Chinese,,0,0,0,,开始说话，然后等说到……
 Dialogue: 0,0:09:12.13,0:09:15.86,English,,0,0,0,,the first third, say, his thoughts had gone past that now.
-Dialogue: 0,0:09:12.13,0:09:15.86,Chinese,,0,0,0,,……比如三分之一的时候，他的思想已经走得很远了
+Dialogue: 0,0:09:12.13,0:09:15.86,Chinese,,0,0,0,,比如三分之一的时候，他的想法已经走得很远了
```
Comment (jyhi) [inline ForEveryone.Net.ass]: 感觉“思想”要好些

**Comment by jyhi** [comment]
I should not use "fix" in the commit message.

**Comment by jyhi** [comment]
The first version is released with tag [bilibili-v1](https://github.com/lmy441900/foreveryone.net-zh_CN-sub/releases/tag/bilibili-v1) and is published. Closing this PR and further discussion should open a new one.

**Pull request closed without merging**